### ISSUE TITLE
Feature request: Brew package

### ISSUE THREAD

**Issue by e1nh4nd3r** [lukakerr/regxr#9]
Self-explanatory.  Would be a super-easy way to get updates and releases.

**Comment by jenhsun** [comment]
@e1nh4nd3r Agree. Hopefully this tool could add to brew-cask. Here is the guide.
<https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md>

**Comment by lukakerr** [comment]
Great idea. I’ll get onto this ASAP
